src/transcoding/codec/codecs/libs/vaapi.c

```diff
@@ -22,6 +22,39 @@
 #include <fcntl.h>
 #include <sys/ioctl.h>
 
+#include "idnode.h"
+#include "htsmsg.h"
+
+/* hts ==================================================================== */
+
+static htsmsg_t *
+platform_get_list( void *o, const char *lang )
+{
+    // 0=Unconstrained 1=Intel 2=AMD
+    static const struct strtab tab[] = {
+        { N_("Unconstrained"),  0 },
+        { N_("Intel"),          1 },
+        { N_("AMD"),            2 },
+    };
+    return strtab2htsmsg(tab, 1, lang);
+}
+
+static htsmsg_t *
+rc_mode_get_list( void *o, const char *lang )
+{
+    // -1=skip 0=auto 1=CQP 2=CBR 3=VBR 4=ICQ 5=QVBR 6=AVBR
```

Comment:
No need to add this as a comment as the table is just below this.

Reply:
done